### ISSUE TITLE
caed not talking to JACK-1.9.14

### ISSUE THREAD

**Issue by hgshoggins** [ElvishArtisan/rivendell#569]
Using Rivendell suite version `2.19.3` here.
With a recent upgrade of a test machine to the latest Fedora 32, shipping with JACK version `1.9.14`, `caed` is not able to connect to the server:
```
$ caed -d
QSettings::sync: filename is null/empty
Cannot connect to server socket err = Aucun fichier ou dossier de ce
type
Cannot connect to server request channel
jack server is not running or cannot be started
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1,
skipping unlock
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1,
skipping unlock
unable to communication with JACK server
caed: unable to communicate with JACK server
JACK general failure
caed: JACK general failure
no connection to JACK server
caed: no connection to JACK server
caed: Found TwoLAME encoder library, MPEG L2 encoding supported
caed: Found MAD decoder library, MPEG L2 decoding supported
caed: using realtime scheduling, priority=8
caed: cae started
caed: cae saveState(), set restart hint to Never
```
(the french `Cannot connect to server socket err = Aucun fichier ou dossier de ce type` must be translated to `No such file or directory`)

With a downgrade to JACK-1.9.12 which is shipped with Fedora 31, it works again:
```
$ caed -d
QSettings::sync: filename is null/empty
caed: connected to JACK server
caed: Found TwoLAME encoder library, MPEG L2 encoding supported
caed: Found MAD decoder library, MPEG L2 decoding supported
caed: using realtime scheduling, priority=54
caed: cae started
caed: cae saveState(), set restart hint to Never
```
NOTE: I performed all these tests with a full cleanup and recompiling of the whole Rivendell suite to ensure, if anyhow necessary, that the appropriate libraries were linked.

Could this also affect version `3.x` ?

**Comment by workino** [comment]
Hello,
Check the caed and jack  user. 
They need to run under the same user otherwise caed cannot talk with jack 

Regards
Gabriele

**Comment by hgshoggins** [comment]
Yup, all running under the same user of course! The only changing element here is the JACK version.

**Comment by hgshoggins** [comment]
After having recompiled and recompiled again, I still had not luck.
Then today I simply decided to add a bit of debug in `cae_jack.cpp`, so I added a line with `LogLine` just to see in which condition it fell when trying to connect to a JACK server, if there was a `jackServerName` defined or not (line `462`). So I edited the file, executed `make && make install` in `rivendell/cae/`, hoping to see something new.
And without any explanation, it worked, connected perfectly to the `default` JACK server.

It puzzles me because the `make && make install` process, along with a complete re-download of the source and a `make uninstall` before, making sure that everything Rivendell-related in `/usr/local/bin` had been removed, never worked.

And now, just by changing this and specifically `make` and `make install` in this directory, it works. I can't seem to find a good explanation, but hey! Problem solved.
